packages/docs/src/snippets/js/webpack_sass_variables.txt

```diff
@@ -1,50 +1,52 @@
 // webpack.config.js
 
 module.exports = {
-  rules: [
-    // SASS has different line endings than SCSS
-    // and cannot use semicolons in the markup
-    {
-      test: /\.sass$/,
-      use: [
-        'vue-style-loader',
-        'css-loader',
-        {
-          loader: 'sass-loader',
-          // Requires sass-loader@^7.0.0
-          options: {
-            // This is the path to your variables
-            data: "@import '@/styles/variables.scss'"
+  module: {
+    rules: [
+      // SASS has different line endings than SCSS
+      // and cannot use semicolons in the markup
+      {
+        test: /\.sass$/,
```

Comment:
In other problem, Is is OK?

"test" is  `/\.sass$/`, but `@import '....scss'.` The extension is different.

Reply:
I don't understand.

Reply:
@johnleider Sorry. I'd like to say the extensions are differrent as follows. The extensions are .sass in "test" and .scss in ".options.data"

<img width="418" alt="Screen Shot 2020-01-23 at 8 10 59" src="https://user-images.githubusercontent.com/10933561/72942949-1d1df200-3db8-11ea-9bef-f937e71ce32a.png">